backend/src/main/java/com/woowacourse/kkogkkog/presentation/MemberController.java

```diff
@@ -19,15 +22,23 @@ public MemberController(MemberService memberService) {
         this.memberService = memberService;
     }
 
+    @GetMapping
+    public ResponseEntity<SuccessResponse<List<MemberResponse>>> showAll() {
+        return ResponseEntity.ok(new SuccessResponse<>(memberService.findAll()));
+    }
+
     @GetMapping("/me")
     public ResponseEntity<MemberResponse> showMe(@LoginMember Long id) {
         MemberResponse memberResponse = memberService.findById(id);
 
         return ResponseEntity.ok(memberResponse);
     }
 
-    @GetMapping
-    public ResponseEntity<SuccessResponse<List<MemberResponse>>> showAll() {
-        return ResponseEntity.ok(new SuccessResponse<>(memberService.findAll()));
+    @PutMapping("/me")
+    public ResponseEntity<Void> updateMe(@LoginMember Long id,
+                                         @RequestBody MemberUpdateMeRequest memberUpdateMeRequest) {
```

Comment:
```suggestion
                                         @RequestBody MemberUpdateRequest memberUpdateRequest) {
```

위 처럼 수정하는건 혹시 어떤가요!! 🙋‍♂️